backend/moonstream/data.py

```diff
@@ -94,8 +94,8 @@ class EthereumTransaction(BaseModel):
     gas: int
     gasPrice: int
     value: int
-    from_address: str = Field(alias="from")
-    to_address: Optional[str] = Field(default=None, alias="to")
+    from_address: str = Field(alias="from_address")
+    to_address: Optional[str] = Field(default=None, alias="to_adress")
```

Comment:
Typo in alias - `to_adress` -> `to_address`. Question about alias still remains.